### ISSUE TITLE
[pyos][bug] pycodestyle detects block params with `E265`

### ISSUE THREAD

**Issue by sneakers-the-rat** [lintquarto/lintquarto#138]
part of: https://github.com/pyOpenSci/software-submission/issues/257#issuecomment-4523576886

pycodestyle is flagging this:

```
#| title: "Mortgage Originations per Capita"  # noqa: E265,E501
#| height: 50%  # noqa: E265,E501
```

like 

```
examples/mortgage-dashboard/index.qmd:77:1: E265 block comment should start with '# '
examples/mortgage-dashboard/index.qmd:78:1: E265 block comment should start with '# '
```

apparently it doesn't read the noqa markers? 

tagging as a bug because this would make it impossible to use that rule with pycodestyle
